Control.cpp

```diff
@@ -45,61 +50,284 @@ double Control::getAngleError(const double current_angle_x, const double desire)
   return error;
 }
 
-double Control::getErrorUltrasonic(const double current_distance, const double desireUltrasonic) {
-  double error = current_distance - desireUltrasonic;
-  return error;
+double Control::getErrorUltrasonic(const double current_distance, const double desire_ultrasonic) {
+  return(current_distance - desire_ultrasonic);
 }
 
 void Control::getPwm(double &speed) { 
-  if (speed < kLimit_inf_pwm) {
-    speed = kLimit_inf_pwm;
+  if (speed < Common::kLimit_inf_pwm) {
+    speed = Common::kLimit_inf_pwm;
   }
   
-  else if (speed > kLimit_sup_pwm) {
-    speed = kLimit_sup_pwm;
+  else if (speed > Common::kLimit_sup_pwm) {
+    speed = Common::kLimit_sup_pwm;
   }
 }
 
 double Control::getNewDesireLeft(double new_desire) {
   new_desire = new_desire - kDegrees90;
   if (new_desire < 0) {
-    new_desire = new_desire + kDegrees360;
+    new_desire += kDegrees360;
   }
   return new_desire;
 }
 
 double Control::getNewDesireRight(double new_desire) {
-  new_desire = new_desire + kDegrees90;
+  new_desire += kDegrees90;
   if (new_desire > kDegrees360) {
-    new_desire = new_desire - kDegrees360;
+    new_desire -= kDegrees360;
   }
   return new_desire;
 }
 
 bool Control::detectRamp() {
-  return (bno_.getAngleY() < -kLimitInfDegrees 
-  && bno_.getAngleY() > -kLimitSupDegrees 
-  && bno_.getAngleZ() < kRangeAngleZ 
-  && bno_.getAngleZ() > -kRangeAngleZ);
+  return (bno_->getAngleY() < -kLimitInfDegrees 
+  && bno_->getAngleY() > -kLimitSupDegrees 
+  && bno_->getAngleZ() < kRangeAngleZ 
+  && bno_->getAngleZ() > -kRangeAngleZ);
```

Comment:
See if you can indent this properly. Visual Studio Code should have a format option. Look for it and see if you can use it.